nilearn/image/resampling.py

```diff
@@ -185,7 +186,7 @@ def get_mask_bounds(img):
 
     """
     img = _utils.check_niimg(img)
-    mask = img.get_data()
+    mask = (img.get_data() != 0.)
```

Comment:
> We have to ensure that the mask is bool.

I think that sklearn.utils.fixes.as_type(mask, np.bool) would be more
memory-friendly.